.gitignore

```diff
@@ -11,4 +11,6 @@ gradle-app.setting
 !gradle-wrapper.jar
 
 # Cache of project
-.gradletasknamecache
\ No newline at end of file
+.gradletasknamecache
+
+src/main/resources/application.properties
```

Comment:
currently the local application.properties file has the username and password information.  I guess we could just export these variables locally instead of having an application.properties file?

Reply:
Oh I see. I saw that y'all were using `Environment` so I assumed that the username and password were provided via environment variables.

It is conventional to commit the `application.properties` or `application.yml` file in a Spring project. Secrets (vs configuration) are using provided via other mechanisms. That could be environment variables or via a property that is provided at runtime.

Reply:
Sounds good